airmozilla/new/templates/new/_email_new_event.html

```diff
@@ -23,6 +23,19 @@
   Video privacy: <b>{{ event.get_privacy_display() }}</b>
 </p>
 
+<p> 
+  Channels:
+  {% if event.channels.all().count() > 0 %} 
```

Comment:
There's no need to compare a count. If it's anything other than `0` it's considered true. 
A much more efficient way to ask the same question is to use `{% if event.channels.all().exists() %}`.